c7n/resources/route53.py

```diff
@@ -613,3 +614,82 @@ def process(self, resources, event=None):
                 results.append(resource)
 
         return results
+
+
+@resources.register('readiness-check')
+class ReadinessCheck(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'route53-recovery-readiness'
+        arn_type = 'readiness-check'
+        enum_spec = ('list_readiness_checks', 'ReadinessChecks', None)
+        name = id = 'ReadinessCheckName'
+        universal_taggable = object()
```

Comment:
afaics we're not using universal (aka resource group tagging) vs manual use of the service's tag api,  in which case this attribute should be omitted.

Reply:
@kapilt , Thanks for the comment. I removed that universal_taggable attribute.